QualityControl/public/object/virtualTable.js

```diff
@@ -29,30 +29,38 @@ export default function virtualTable(model, location = 'main') {
   return h('.flex-grow.flex-column', {
   }, [
     location !== 'side' && tableHeader(),
-    h(
-      '.scroll-y.animate-width',
-      tableContainerHooks(model),
+    model.object.searchResult.length === 0 ? tableEmpty() :
```

Comment:
Keep in mind that `virtualTable` is a reused component by other plages.
If you look one line above, you will see that the `tableHeader` is added already under a condition which checks the page location. 

Now because you add another table, on object tree page, we will have two tables.

The standard is that the `No objects found for this search` message in case of no data matching, is that it should be added as a row and not as a header (check screenshot to see how it looks with these changes on the object tree page)

<img width="1047" alt="image" src="https://github.com/user-attachments/assets/a264b865-c0ff-4942-9913-fbe57eb49282">